doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -111,11 +111,15 @@ Note: with an underlying IaaS this is possible, but then it introduces (undesira
 <a name="6.2.9"></a>
 ### 6.2.0 User namespaces in Kubernetes
 
-> **Related requirements:** `e.man.004`, `sec.gen.006`
+> **Related requirements:**
+| Reference | Requirement |
+|---|---|
+| e.man.004 | Capability to isolate resources between tenants |
+| sec.sys.007 | The Platform must implement controls enforcing separation of duties and privileges, least privilege use and least common mechanism (Role-Based Access Control). |
 
 > **Baseline project:** _Kubernetes v1.20_
 
-> **Gap description:** Kubernetes does not have a support for user namespaces to provide namespace scoped user ID-s (UID), therefore when an application in a container requires system user privileges the container either needs to run in privileged mode or the infrastructure provides random system UID-s. Scrambling UID-s results in errors when the application needs to set kernel capabilities (e.g.: in case of VLAN trunking) or when a Pod shares data with other Pod-s via persistent storage, therefore none of these solutions are secure or error prone, therefore they are not advised. Support for proper user namespaces in Kubernetes is [under discussion](https://github.com/kubernetes/enhancements/pull/2101).
+> **Gap description:** Kubernetes does not support namespace scoped user IDs (UIDs). Therefore, when a container-based application in a requires system privileges the container either needs to run in privileged mode or the infrastructure needs to provide random system UIDs. Randomised UIDs result in errors when the application needs to set kernel capabilities (e.g.: in case of VLAN trunking) or when a Pod shares data with other Pods via persistent storage. The "privileged mode" solution is not secure while "random UIDs" is error prone, and therefore these techniques are advised for use. Support for proper user namespaces in Kubernetes is [under discussion](https://github.com/kubernetes/enhancements/pull/2101).
```

Comment:
correct typo pointed out by @karinesevilla: " when a container-based application in a requires system privileges " -> 'in a' should be removed; should also remove "in"
typo missing "not" in "... and therefore these techniques are advised for use"; suggest slight modification
```suggestion
> **Gap description:** Kubernetes does not support namespace scoped user IDs (UIDs). Therefore, when a container-based application requires system privileges the container either needs to run in privileged mode or the infrastructure needs to provide random system UIDs. Randomised UIDs result in errors when the application needs to set kernel capabilities (e.g.: in case of VLAN trunking) or when a Pod shares data with other Pods via persistent storage. The "privileged mode" solution is not secure while "random UID" solution is error prone, and therefore these techniques should not be used. Support for proper user namespaces in Kubernetes is [under discussion](https://github.com/kubernetes/enhancements/pull/2101).
```